### ISSUE TITLE
Disabling Imprint Vibration doesn't work on OxygenOS

### ISSUE THREAD

**Issue by kocane** [GravityBox/GravityBox#1978]
Using OnePlus 6 with OxygenOS 9.0.4 and GravityBox 9.0.0-beta-10 I can't disable the Imprint vibration on succesful/failed fingerprint authentication.

**Comment by C3C0** [comment]
Yes, I know about this issue. Methods  that handle vibrations cannot be hooked by edxposed for some reason and I currently don't have any alternate solution.

**Comment by kocane** [comment]
> 
> 
> Yes, I know about this issue. Methods that handle vibrations cannot be hooked by edxposed for some reason and I currently don't have any alternate solution.

Oh okay :-( Might be a dumb question but how does the notification control work then?

**Comment by C3C0** [comment]
Methods that handle fingerprint event vibrations have nothing to do with other types of vibrations
